presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.h

```diff
@@ -34,6 +35,26 @@ void unregisterPrestoToVeloxConnector(const std::string& connectorName);
 const PrestoToVeloxConnector& getPrestoToVeloxConnector(
     const std::string& connectorName);
 
+velox::dwio::common::FileFormat toVeloxFileFormat(
```

Comment:
Might be worth moving all the Iceberg related processing that transforms to Hive structures in the HivePrestoToVeloxConnector classes as well.